test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -241,8 +241,12 @@ def test_cray_login_node_memory_unit(self):
             self.assertFalse(True,
                              "Mom node %s doesn't exist on pbs server"
                              % (mom_id))
+        # List of resources to be ignored while comparing.
+        ignr_rsc = ['license']
 
         for rsc, val in pbs_node.iteritems():
+            if rsc in ignr_rsc:
+                continue
```

Comment:
instead of ignoring the license attribute, why don't we only check for mem?

Reply:
Hi @prakashcv13 ,
     I feel that testing for all the node attributes add more value to this test case. Since alps re-initialzation of cray_login  vnode also effects other node attributes.

Reply:
it does make sense if we do not have exceptions like we have for the license attribute. Are we planning to add it back once the license issue is resolved?

Reply:
If it's decided that the difference in license attribute is a bug, then we can modify the same test case to validate the fix, otherwise we will keep it as is.

Reply:
Okay.